### ISSUE TITLE
additional property aside from name

### ISSUE THREAD

**Issue by vvavepacket** [arjunmehta/node-georedis#20]
Hi,

GeoRedis feature currently utilizes 3 properties: latitude, longitude, and name. When adding a location, is it possible to add additional key value pairs aside from name itself? Let us say we are adding name, address, etc. Is it possible? If not, what are best workaround / practices?

**Comment by vvavepacket** [comment]
closed... duplicate found. https://github.com/arjunmehta/node-georedis/issues/6

**Comment by arjunmehta** [comment]
@vvavepacket Yeah, best bet is to have another hash store based on names. Redis stores everything as strings or numbers, but you can use `hmset`/`hmget` at least have some metadata. ie. `client.hmset("mylocationname", { a: 1, b: 2, c: 'xxx' })`

Redis isn't super great for relational queries or storing complex objects at a key. Though, I actually think it could be a nice convenience to be able to store simple metadata for a location. It would be probably be limited to a shallow object of either strings and numbers.

I would like to leave this open as a feature request but the API might look like:

```JavaScript
geo.addLocation('locationname', position, {
    address: '265 Homer Lane',
    city: 'Lalaland, CA',
    rank: 3
})
```

OR

```JavaScript
geo.addLocation('locationname', {
  latitude: lat, 
  longitude: lon,

  address: '265 Homer Lane',
  city: 'Lalaland, CA',
  rank: 3
})
```

**Comment by saitolabs** [comment]
+1

**Comment by saitolabs** [comment]
maybe instead of:

```
geo.addLocation('locationname', position, { 
    address: '265 Homer Lane',
    city: 'Lalaland, CA',
    rank: 3
})
```
we can work with:

```
geo.addLocation('locationname', {
    latitude: 43.6667, 
    longitude: -79.4167,
    address: '265 Homer Lane',
    city: 'Lalaland, CA',
    rank: 3
})
```
latitude -> required
longitude -> required
